yo1k/tic_tac_toe/test/core/test_game.py

```diff
@@ -77,7 +79,8 @@ def test_advance__surrender_action(self):
         expected_state = _new_state(
                 board=Board([[None, None, None], [None, None, None], [None, None, None]]),
                 phase=Phase.OUTROUND,
-                player_o=_new_player(mark=Mark.O, wins=1))
+                player_o=_new_player(mark=Mark.O, wins=1),
+                required_ready=set(range(2)))
```

Comment:
> P.P.S. set(range(2)) is the default value, why do you set it explicitly?

`set(range(2))` is not default value of the `_new_state` function, `set()` (or empty set) is the real value. The `_new_state` function has default state corresponding to `Phase.INROUND` and because of that I made `required_state` as empty set, accordingly.